wof/examples/configs/timeseries/timeseries.conf

```diff
@@ -0,0 +1,19 @@
+# description of the script purpose
```

Comment:
Filename `timeseries.conf` is too general. I think best way is to make this `wofpy.conf` to server multiple dao also.